scripts/version

```diff
@@ -61,8 +65,8 @@ if echo $TAG | grep -q dirty; then
     TAG=dev
 fi
 
-if [ -n "$DRONE_TAG" ]; then
-    TAG=$DRONE_TAG
+if [ -n "$GIT_TAG" ]; then
+    TAG=$GIT_TAG
 fi
```

Comment:
```suggestion
```
I think this is redundant

Reply:
let me check first if this is really redundant.

Reply:
removed